search/Dockerfile

```diff
@@ -1,14 +1,13 @@
-FROM node:10
-RUN mkdir /code
-WORKDIR /code
-COPY package.json yarn.lock /code/
-RUN ["yarn", "install"]
+# FROM node:10
+# RUN mkdir /code
+# WORKDIR /code
+# COPY package.json yarn.lock /code/
+# RUN ["yarn", "install"]
+# EXPOSE 3000
+# CMD ["yarn", "start"]
```

Comment:
Where is reactivesearch now being launched?

Reply:
Now it's more like a production version, as we have to run `yarn run build` to build the static files and docker will serve the build folder onto nginx server automatically. So maybe it's better to have a development branch, where docker-compose starts the react server with the volume? I'm not sure what's the best way to manage this.

Reply:
Okay, but then `yarn build` should be in the Docker build?

Reply:
yes, it might be nice to have a dev version where we just use another pass-through from django???? Maybe we can't have our cake and eat it too.

Reply:
reactivesearch now is based on a nginx image, so probably can't run that yarn run build inside either docker-compose.yml nor the Dockerfile, as those will be run inside the nginx container. With docker 17.0.6, it can be done by multi stage build, where it creates a temporary container of node image and copy the src code and run `yarn run build` inside and then copy the build folder from this container to the nginx container and serve it.

Reply:
Annoying that I can't easily see which version introduced secrets to docker-compose :(